app/services/fetch_remote_status_service.rb

```diff
@@ -39,9 +39,17 @@ def extract_author(url, xml)
 
     Rails.logger.debug "Going to webfinger #{username}@#{domain}"
 
-    return FollowRemoteAccountService.new.call("#{username}@#{domain}")
+    account = FollowRemoteAccountService.new.call("#{username}@#{domain}")
+
+    return nil if confirmed_domain?(domain, account)
```

Comment:
This doesn't quite read right to me, return `nil` if the domain is confirmed? If it's correct, a clarifying comment could be helpful 👍